modules/gothamadsBidAdapter.js

```diff
@@ -94,9 +109,23 @@ export const spec = {
         source: {
           tid: bidRequest.transactionId
         },
+        regs: {
+          coppa: config.getConfig('coppa') === true ? 1 : 0,
+          ext: {}
+        },
         tmax: bidRequest.timeout,
         imp: [impObject],
       };
+      if (bidRequest) {
```

Comment:
You don't actually need this `if`.  
In couple of lines above you are already retrieving properties from `bidRequest` object (for example line 116: `bidRequest.timeout`).  
So, in this line `bidRequest` would **always** evaluate to true.

Reply:
@FilipStamenkovic  Hello! Thank you! I have fixed adapter according to your comments!